### DISCUSSION TITLE
Drag and Drop entity passing but certain values aren't

### DISCUSSION THREAD

**Discussion by HALAYUDHA-dev** [tokarchyn/react-easy-diagram#14] [General]
So I have been using Drag and Drop implementation, and creating a custom node. So the type of new node is caught but label is not passed to the new created node.

Diagram Context

**<DiagramContext
      settings={{
        nodes: {
          components: {
            batch_node: NodeTemplate,
          },
        },
        ports: {
          components: {
            batch_to_job: createLinkDefault
          },
        },
      }}
      initState={{
        nodes: [
          {
            id: "node_1",
            label: "Node 1",
            position: [300, 300],
            type: "output_horizontal",
          },
        ],
      }}
    >
      <ContainerDiv>
        <DigramInner />
        <TreeMain />
      </ContainerDiv>
    </DiagramContext>** 

DragAndDrop Item

**<DraggableDiv>
      <DragIcon color="#C3C3C3" />
      <DragAndDropItem
        draggable={node.batchName}
        onDrop={createNodeOnDrop({ type: "batch_node", label: "hello"})}
      >
        <NodeText>{node.batchName}</NodeText>
      </DragAndDropItem>
    </DraggableDiv>**

custom node

**export function NodeTemplate(props) {
  return (
    <div
      className={"react_fast_diagram_NodeDefault"}
      style={{
        width: 200,
        height: 30,
      }}
    >
      {console.log(props.entity.label)}
      {props.entity.label}
    </div>
  );

}**

but we can only catch the type but not the passed label
![Screenshot 2022-05-09 144031](https://user-images.githubusercontent.com/57007177/167378473-e2b80732-703e-49ff-b2aa-add6c6ac4a2f.png)

**Comment by tokarchyn** [comment]
Thank you for the interest to this package🙂. The new version `0.9.3` has been released to fix the problem you shared.
